README.md

```diff
@@ -176,8 +151,12 @@ The edge build is generated for each push to the `dev` branch as a part of the T
 
 
 ## Developer Setup
-If you would like to setup a development environment and contribute to the CLI, see
-[Configuring Your Machine](https://github.com/Azure/azure-cli/blob/dev/doc/configuring_your_machine.md).
+
+If you would like to setup a development environment and contribute to the CLI, see:
+
+[Configuring Your Machine](https://github.com/Azure/azure-cli/blob/dev/doc/configuring_your_machine.md)
+
+[Authoring Command Modules](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules)
```

Comment:
Added this so once someone has set up their environment they can proceed to writing commands.